neurom/core/morphology.py

```diff
@@ -538,7 +538,7 @@ def __repr__(self):
 class Morphology:
     """Class representing a simple morphology."""
 
-    def __init__(self, filename, name=None, process_subtrees=False):
+    def __init__(self, filename, name=None, *, mutable=None, process_subtrees=False):
```

Comment:
While I can see the point of adding the option to load_morphology, adding it here it will only complicate things. Frankly, The Morphology wrapper shouldn't allow passing a file path to begin with, given that the entry point of loading a morphology are the io functions...

Reply:
Indeed if we can't pass a file path it's not needed. But since it is allowed I think we should allow to choose whether the loaded morph is mutable or not.

Reply:
How about cleaning it up by allowing to pass only morphio morph instances and let io deal with loading from file. This way io and wrapper will be decoupled instead of passing here options that are for io.

Reply:
I think it makes sense!